src/App.js

```diff
@@ -10,40 +15,165 @@ class App extends React.Component {
       // Set default value of card deck to new shuffled deck
       cardDeck: makeShuffledDeck(),
       // currCards holds the cards from the current round
-      currCards: [],
+      round: 0,
+      player1: { winGame: 0, score: 0, result: "", currCard: {} },
+      player2: { winGame: 0, score: 0, result: "", currCard: {} },
+      restart: "",
+      dealButton: <button onClick={this.changeResult}>Deal</button>,
```

Comment:
I think the button should not live in state. Ideally don't place HTML elements here, but only data.